tox.ini

```diff
@@ -53,12 +53,12 @@ deps =
     -rrequirements/edx/base.txt
     -rrequirements/edx/development.txt
     # There's 1-2 tests which call a paver command...
```

Comment:
Fine with this PR merging - just wish this comment would have moved along with the paver.txt line.